core/dbt/cli/main.py

```diff
@@ -52,17 +55,30 @@ def cli(ctx, **kwargs):
     """An ELT tool for managing your SQL transformations and data models.
     For more documentation on these commands, visit: docs.getdbt.com
     """
-    incomplete_flags = Flags()
+    flags = Flags(invoked_subcommand=globals()[ctx.invoked_subcommand])
+
+    dbt.tracking.initialize_from_flags(flags)
+    # TODO we need to have config to get the projectID
+    project_id = ""
+    # TODO we need to get the credentials or we need to know that this will make adapter info
+    # not available in invocation start/end events
+    credentials = None
+    ctx.with_resource(dbt.tracking.track_run(project_id, credentials, ctx.invoked_subcommand))
+
+    # Logging
```

Comment:
I think you merged all the changes from my logging branch, not just the changes to flags.
:)

As above, feel free to leave it if you want, but it's all changed since you pulled it in from my branch.